google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java

```diff
@@ -255,13 +255,24 @@ public boolean equals(Object o) {
 
     abstract ImmutableList<FieldReference> getFieldProjections();
 
+    // Whether to select all documents under `parentPath`. By default, only
+    // collections that match `collectionId` are selected.
+    abstract boolean getKindless();
+
+    // Whether to require consistent documents when restarting the query. By
+    // default, restarting the query uses the readTime offset of the original
+    // query to provide consistent results.
+    abstract boolean getRequireConsistency();
```

Comment:
This should technically also not be `get`, but `is` doesn't work so you may want to keep it. I also wonder whether it should be "requires".